app/assets/javascripts/miq_tree.js

```diff
@@ -65,15 +65,12 @@ function miqRemoveNodeChildren(treename, key) {
   }
 }
 
-function miqOnCheckMenuRoles(id) {
-  var nid = id.split('__');
-  if (nid[0] !== 'r') {
```

Comment:
If this were about reports, it would be `rep` or `rr`.

`r` is a ResourcePool.

The condition has been part of the `miqMenuEditor` (`vmdb/app/assets/javascripts/cfme_dynatree.js`) function since forever, so maybe that changed at some point, or it was always wrong.

Not seeing any indication that `TreeBuilderMenuRoles` ever dealt with ResourcePool, LGTM :+1: